backend/typescript/types.ts

```diff
@@ -83,6 +84,11 @@ export type CampDTO = {
   active: boolean;
 };
 
+export type GetCampDTO = Omit<
+  CampDTO & BaseCampDTO,
+  "baseCamp" | "campers" | "waitlist" | "camps"
```

Comment:
Do we need to return "campers" and "waitlist"? Or will this only be required when getting the csv or a specific camp

Reply:
I remember that I discussed the same question with @annechung, and it was decided that campers and waitlist are not to be returned, but just to confirm again, @annechung do we need "campers" and "waitlist" in get camp service?

Reply:
yepp that's right, we only need to get campers and waitlist for specific camps